### PR TITLE
4481/4 data

### PR DESCRIPTION
**Pull request by chris-wm**
Add job entity

### CODE REVIEW

File: dataTypes/dataTypes.go [hunk 1]
```diff
@@ -14,24 +15,40 @@ type Field struct {
...
+	var fieldInfo FieldMap
+	err := json.Unmarshal(fieldMap, &fieldInfo)
+	if err != nil {
+		panic(err)
+	}
+
+	var dataInfo map[string] string
+	err = json.Unmarshal(data, &dataInfo)
+	if err != nil {
+		panic(err)
```
Comment (krisrybak) [inline dataTypes/dataTypes.go:39]: To quote Alex "Panic is a sackable offence ..." That will terminate program (service). Better to log and (or) return error

File: dataTypes/example.go [hunk 2]
```diff
@@ -2,40 +2,31 @@ package dataTypes
...
+	rawData := json.RawMessage(dataJSON)
+	data, err := rawData.MarshalJSON()
+	if err != nil {
+		panic(err)
+	}
+
+	rawField := json.RawMessage(fieldMapJSON)
+	fieldMap, err := rawField.MarshalJSON()
+	if err != nil {
+		panic(err)
```
Comment (krisrybak) [inline dataTypes/example.go:24]: 😱 panic and run for cover ... program terminated ;)

File: dataTypes/example.go [hunk 3]
```diff
@@ -2,40 +2,31 @@ package dataTypes
...
+	}
+	
+	ToElectiveStruct(fieldMap, data)
+
+	//// Run App Migrations
+	//ret := ToElectiveStruct(fieldMap, data)
+	//fmt.Print(ret)
 }
```
Comment (krisrybak) [inline dataTypes/example.go:32]: Print and empty line

File: dataTypes/example.go (hunk 2 above)
Comment (chris-wm) [reply dataTypes/example.go]: yeah this was work in progress, panic removed in later MR

**Review by krisrybak — APPROVED** [review]

**Pull request merged**